changelog.txt

```diff
@@ -1,5 +1,17 @@
 *** Changelog ***
 
+## 4.23.0 - 2024-04-04
+### Added
+- Sensei pro upsell task in Sensei Home [#7562](https://github.com/Automattic/sensei/pull/7562)
+
+### Deprecated
```

Comment:
It's the first time we'll introduce this title in this changelog. I think it's totally fine, but I'd like to highlight with this comment just because it's the first time, and if someone realizes something that could break.